README.md

```diff
@@ -365,6 +365,49 @@ private Binding<HttpStatus> problems(final Condition<HttpStatus> condition) {
 }
 ```
 
+#### Template Routing
+
+Routing and response handling is sometimes pretty similar for many request, and can even be handle using the same
+defined routing logic for different requests. To enable reuse of this routing logic, Riptide supports the creation
+of nested *template routing*, that can be reused for dispatching different request. The following example defines
+a nested hierarchy of routing templates:
+
+```java
+static final Router<MediaType> CONTENT_ROUTER = Router.create(contentType(),
```

Comment:
> Ok, but do we have the evidence that this is a common case?

At least I would do it in this way.

> Do you intend to share routing trees across gateways?

It would be possible if APIs of the gateways follow the same rules/patterns as suggested by our API guidelines.

> Or inside a gateway across different kind of HTTP requests to the same remote service?

Yes, I would do. For most APIs you pretty much expect the same behavior for each response. Even the differences between GET, PUT, and POST are mostly very small and can be handled by the same set of rules:

Status based routing is taken care of by the status router, content based routing is taken care of by the content based routing, etc.